src/frameworks/ez.md

```diff
@@ -18,4 +18,129 @@ In particular, see:
+
+#### Solr specificity
+
+Solr is fully functional with eZ Launchpad but its auto-configuration for Platform.sh is not currently managed. You will have to set it up manually following the current documenation here: https://github.com/ezsystems/ezplatform/blob/master/.platform/services.yaml#L37.
+
+Actions needed are:
+
+ * generare the Solr configuration thanks to the script provided by eZ Systems
+ * put the result in the `.platform` at the root of your project
+ * add the service in the `.platform/services.yaml`
+ * add the relationship in the `.platform.app..yaml`
+
+#### Environment variables (optional)
+
+eZ Launchpad allows you to define environment variables in the `provisioning/dev/docker-compose.yml` file. You may use that to set [Platform.sh variables](/development/variables.md) to match Platform.sh environments so that you can keep your environment behavior in sync.
+
+Such variables have to be set in the `engine` container.
+
+```yaml
+# provisioning/dev/docker-compose.yml
+engine:
+    environment:
+        - ASIMPLEVARIABLE=avalue
+        - PLATFORM_RELATIONSHIPS=A_BASE64_ENCODED_VALUE
+```
+
+### Local development with Platform.sh
+
+Thanks to eZ Launchpad we are able to be work 100% locally: [untethered](/gettingstarted/local/untethered.md). We have the whole project working offline on our local machine.
+
+> Platform.sh also provides a smooth SSH tunnels integration described in the 
+[tethered](/gettingstarted/local/tethered.md) page. 
+
+Local services are ensured by the Docker stack but there are minimum day-to-day tasks that you might need with Platform.sh.
+
+Main ones being:
+
+- **Downstream database synchronization**: Getting it from the remote to the local.
+- **Downstream file storage synchronization**: Getting it from the remote to the local.
+
+To help you with that, Platform.sh provides a CLI that you probably already have, if you don’t
+```bash
+curl -sS https://platform.sh/cli/installer | php
+```
+
+> More information: [Command Line Interface](/gettingstarted/cli.md)
+
+
+Combine together, eZ Launcphad and Platform.sh CLI make those actions straight forward and simple.
+
+#### Database and storage synchronization
+
+```bash
+platform db:dump --gzip -f ezplatform.sql.gz -d data/ -y
+platform mount:download -m ezplatform/web/var --target=ezplatform/web/var/ -y
+~/ez/importdata
+```
+
+The two first lines get the remote database and storage from the remote environment and stores it locally in `data/`. The third tells to eZ Launchpad to import those data in the Docker stack.
+
+
+> Regarding the storage(images and files) synchronization: **it is optional**, eZ Platform provides a 
+[placeholder generator mechanism](https://doc.ezplatform.com/en/latest/guide/images/#setting-placeholder-generator)
+which allows you to forget about the real images for your local.
+
+#### Other automations
+
+eZ Launchpad wraps Docker, then it allows you to stay on the host and execute actions in containers. 
```

Comment:
This sentence doesn't parse.  I think you mean: "eZ Launchpad serves as a wrapper that allows you to run console commands from within the container without logging into it explicitly.  For example:"